src/main/java/io/kokuwa/maven/helm/AbstractHelmMojo.java

```diff
@@ -124,10 +124,20 @@
 	String projectVersion;
 
 	private Clock clock = Clock.systemDefaultZone();
+	private StripSensitiveDataLog log;
 
 	@Override
 	public Log getLog() {
-		return new StripSensitiveDataLog(super.getLog());
+		if (log == null) {
+			log = new StripSensitiveDataLog(super.getLog());
+		}
+		return log;
+	}
+
+	@Override
+	public void setLog(Log log) {
+		super.setLog(log);
+		this.log = new StripSensitiveDataLog(log);
 	}
```

Comment:
Now I see 

```
public void setLog(Log log) {
    super.setLog(new StripSensitiveDataLog(log)); 
}
```

should be enough, in this case additional fields and override of `getLog`  is not needed